### PR TITLE
Fixed flows_compare

### PR DESCRIPTION
**Pull request by angelika-mamchych**
- fixed compare page
- fixed broken redirect
- added margin between flows_compare
- fixed db weird issue

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -56,23 +45,17 @@ def flow_item_result(flow_id):
...
     total_amount = float(request.form['totalamount'].strip())
     fees = []
+    total_left = total_amount
     for step in flow.steps:
         if step.type == 'fixed':
             fee = step.amount
-            # all our fee equals step.amount
-            # in the end they all will be in
-            # the list fee=[] and total_left=total_amount-fee[]
+            # all our fee equals step.amount in the end they all will be in the list fee=[] and total_left=total_amount-fee[]
```
Comment (oleksii-dmytrenko) [inline app.py]: the comment seems no longer relevant, is it?

File: app.py [hunk 2]
```diff
@@ -91,53 +74,48 @@ def show_flows():
...
-    flows = m.Flow.query.filter(Flow.id.in_(ids)).all()
+    flows = m.Flow.query.filter(m.Flow.id.in_(ids)).all()
     return render_template("flows_compare.html", flows=flows)
 
 
 @app.route("/flows_compare", methods=['POST'])
 def flow_compare_result():
     ids = request.args.getlist('ids')
-    flows = m.Flow.query.filter(Flow.id.in_(ids)).all()
+    # dummy_fees = {
```
Comment (oleksii-dmytrenko) [inline app.py]: please remove the dummy shit

File: templates/flow_form.html [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
   class Step extends React.Component {
     render() {
-      const style = {border: "1px dotted blue", padding: "5px", marginBottom: "5px", borderRadius: "5px"};
+      const style = {border: "1px dotted blue", padding: "5px", marginBottom: "5px", marginRight: "15px", borderRadius: "5px"};
```
Comment (oleksii-dmytrenko) [inline templates/flow_form.html]: please revert this

File: templates/flows_compare.html [hunk 4]
```diff
@@ -2,31 +2,22 @@
 
 {% block body %}
 {% for flow in flows %}
-    <div style="float: left">
+    <div style="float: left; margin: 50px">
```
Comment (oleksii-dmytrenko) [inline templates/flows_compare.html:5]: nice 👍

Comment (angelika-mamchych) [reply templates/flows_compare.html:5]: thanks :kiss:

**Pull request merged**